xml/ha_cluster_lvm.xml

```diff
@@ -117,6 +117,24 @@
      otherwise lvmlockd will fail to start.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     On all nodes, check the value of <parameter>use_lvmlockd</parameter>
+     in <filename>/etc/lvm/lvm.conf</filename>:
+     </para>
+     <screen>&prompt.root;grep 'use_lvmlockd =' /etc/lvm/lvm.conf</screen>
```

Comment:
According to the manpage of `lvm.conf`, whitespace is not significant in the config file. This means, a user can add as many spaces between `use_lvmlockd` and equal sign as he or she wants.

Therefore, I would suggest to use `egrep` instead of `grep`:

```suggestion
     <screen>&prompt.root;egrep 'use_lvmlockd\s*=' /etc/lvm/lvm.conf</screen>
```